plugin/session_buffer.py

```diff
@@ -353,9 +354,8 @@ def _do_color_boxes_async(self, view: sublime.View, version: int) -> None:
                 self._if_view_unchanged(self._on_color_boxes_async, version)
             )
 
-    def _on_color_boxes_async(self, view: sublime.View, response: Any) -> None:
-        color_infos = response if response else []
-        self.color_phantoms.update([lsp_color_to_phantom(view, color_info) for color_info in color_infos])
+    def _on_color_boxes_async(self, view: sublime.View, response: List[ColorInformation]) -> None:
+        self.color_phantoms.update([lsp_color_to_phantom(view, color_info) for color_info in response])
```

Comment:
See https://github.com/sublimelsp/LSP/pull/2053#discussion_r972370161

Reply:
Okay, good to know. I added an explicit comment about that, so that the next one who touches this code won't try to do the same again :D